xl-op/digitalai/generated_answers.yaml.tmpl

```diff
@@ -2,6 +2,14 @@ K8sSetup: {{ .K8sSetup }}
 UseCustomNamespace: {{ .UseCustomNamespace }}
 Namespace: {{ .Namespace }}
 ServerType: {{ .ServerType }}
+ImageRegistryType: {{ .ImageRegistryType }}
+{{- if eq .ImageRegistryType "public" }}
+CustomImageRegistryName: {{ .CustomImageRegistryName }}
+{{- end }}
+{{- if eq .ImageRegistryType "private" }}
+CustomImageRegistryName: {{ .CustomImageRegistryName }}
+CustomPrivateImageRegistrySecret: {{ .CustomPrivateImageRegistrySecret }}
+{{- end }}
```

Comment:
Can we just put CustomImageRegistryName and CustomPrivateImageRegistrySecret? Previously a lot of logic in this file caused a lot of bugs. It is hard to maintain the logic afterward.